extension/chrome/elements/composer/composer-size.ts

```diff
@@ -111,14 +111,13 @@ export class ComposerSize extends ComposerComponent {
   }
 
   /**
-* On Firefox, we have to manage textbox height manually. Only applies to composing new messages
-* (else ff will keep expanding body element beyond frame view)
-* A decade old firefox bug is the culprit: https://bugzilla.mozilla.org/show_bug.cgi?id=202081
-*
-* @param updateRefBodyHeight - set to true to take a new snapshot of intended html body height
-*/
+   * Manage textbox height manually. Only applies to composing new messages
+   * (otherwise input_text will keep expanding beyond frame view)
+   *
+   * @param updateRefBodyHeight - set to true to take a new snapshot of intended html body height
+   */
   public setInputTextHeightManuallyIfNeeded = (updateRefBodyHeight: boolean = false) => {
-    if (!this.view.isReplyBox && Catch.browser().name === 'firefox') {
+    if (!this.view.isReplyBox) {
```

Comment:
I figure you want the behavior on chrome/ff more consistent?

I don't like browser-specific code either, but never had this issue on Chrome, ant thought maybe the code is expensive, so on Chrome I let the browser handle it (since it could).

Reply:
I'll try to revert this change and to fix the oveflowing on the CSS side :+1:

Reply:
I'm quote ok with either the way it was or with your change, as long as it's a deliberate choice :+1:

Reply:
I reverted this change.